app/components/planning_applications/immunity_details_assessment_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  class ImmunityDetailsAssessmentComponent < ViewComponent::Base
+    def initialize(planning_application:, evidence_groups:)
+      @planning_application = planning_application
+      @evidence_groups = evidence_groups.map do |group|
+        EvidenceGroupPresenter.new(view_context, group)
+      end
+    end
```

Comment:
I think passing the `evidence_groups` separately might not be needed.  `evidence_groups` are just `planning_application.immunity_detail.evidence_groups` so bypassing only the `planning_application` is good enough, what do you think?

Reply:
Are Presenter classes instantiated automatically? Perhaps I added that unnecessarily in that case.

Reply:
I just tried and I think I do need to instantiate the presenter, otherwise it can't find certain method names